code/src/index.css

```diff
@@ -1,13 +1,216 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+ margin: 0;
+ background: #000;
+ color: #fff;
+ font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Fira Sans,Droid Sans,Helvetica Neue,sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+
+/*** FIRST PAGE STYLING ***/
+.filter-btn-container {
+  height: 50px;
+}
+
+.filter-btn {
+  margin: 10px;
+  padding: 8px;
+  background-color: #fff;
+  border: none;
+  border-radius: 5px; 
+  font-weight: bold;
+  transition: 0.1s ease-in;
+}
+
+.filter-btn:hover {
+  background-color: red;
+  color: #fff;
+}
+
+.movie-container {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.popular-movie {
+  position: relative;
+  color: #fff;
+}
+
+.movie-img {
+  width: 100%;
+}
+
+.movie-details {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 10px;
+  background: rgba(0,0,0,.75);
+  opacity: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.movie-details:hover {
+  opacity: 1;
+}
+
+.movie-title {
+  margin-top: auto;
+  margin-bottom: 0;
+  font-size: 35px;
+}
+
+/*** SECOND PAGE STYLING ***/
+.detail-background {
```

Comment:
When you are re-sizing the screen, the background picture stretches and is not responsive. Try using `object-fit: cover ` as styling to this picture and everything will work fine.